main.py

```diff
@@ -138,3 +144,19 @@ def download(
         return {'message': 'Download sucessful'}
     except Exception as error:  # pragma: no cover
         return {'detail': error}
+
+
+@app.get("/list", response_class=HTMLResponse, tags=['Web UI'], summary='List downloaded files')
+def list_downloads_page(request: Request):
+    download_path = "/downloads"
+    try:
+        files = os.listdir(download_path)
+        file_links = [
+            f'<li class="list-group-item"><a href="/downloads/{file}" download>{file}</a></li>'
```

Comment:
```suggestion
            f'<li class="list-group-item"><a href="/downloads/{file}">{file}</a></li>'
```

I believe it would be interesting to remove the `download` attribute from the `a` tag in the listing, because as we already have the files saved in the mount point mapped to `/downloads`, it is not necessary to download the files that are already saved through the browser.